docs/index.md

```diff
@@ -126,16 +126,16 @@ title: Whiting index
 |  8  | [N3](#n3) [![](w/page79a.gif)][P79] | [N3](#n3) [![](w/page101a.gif)][P101] | [N3](#n3) [![](w/page122a.gif)][P122] | [N3](#n3) [![](w/page141a.gif)][P141] | [N3](#n3) [![](w/page162a.gif)][P162] | [N3](#n3) [![](w/page184a.gif)][P184] | [N3](#n3) [![](w/page206a.gif)][P206] | [N3](#n3)<br> [![](w/page227a.gif)][P227] |
 |  9  | [N3](#n3) [![](w/page80a.gif)][P80] | [N3](#n3) [![](w/page102a.gif)][P102] | [diagr][C9], [N5](#n5):[v][v] [![](w/page123a.gif)][P123] | [diagr][D9] [![](w/page142a.gif)][P142] | [diagr][E9] [![](w/page163a.gif)][P163] | [diagr][F9], [N5](#n5):[k][k] [![](w/page185a.gif)][P185] | [diagr][G9], [N5](#n5):[t][t] [![](w/page207a.gif)][P207] | [diagr][H9], [N5](#n5):[t][t] [![](w/page229a.gif)][P229] |
-|  10 | [N3](#n3) [![](w/page82a.gif)][P82] | [N3](#n3) [![](w/page103a.gif)][P103] | [N3](#n3) [![](w/page124a.gif)][P124] | [N3](#n3) [![](w/page143a.gif)][P143] | [diagr][E10] [![](w/page164a.gif)][P164] | [N3](#n3) [![](w/page187a.gif)][P187] | [diagr][G10], [![](w/page208a.gif)][P208] | [diagr][H10] [![](w/page230a.gif)][P230] |
+|  10 | [N3](#n3) [![](w/page82a.gif)][P82] | [N3](#n3) [![](w/page103a.gif)][P103] | [N3](#n3) [![](w/page124a.gif)][P124] | [N3](#n3) [![](w/page143a.gif)][P143] | [diagr][E10] [![](w/page164a.gif)][P164] | [N3](#n3) [![](w/page187a.gif)][P187] | [diagr][G10] [![](w/page208a.gif)][P208] | [diagr][H10] [![](w/page230a.gif)][P230] |
 |  11 | [diagr][A11], [N5](#n5):[t][t] [![](w/page83a.gif)][P83] | [diagr][B11], [N5](#n5):[t][t] [![](w/page104a.gif)][P104] | [N3](#n3) [![](w/page125a.gif)][P125] | [diagr][D11], [N5](#n5):[wk][wk] [![](w/page144a.gif)][P144] | [diagr][E11], [N5](#n5):[wk][wk] [![](w/page166a.gif)][P166] | [diagr][F11], [N5](#n5):[t][t] [![](w/page189a.gif)][P189] | [val](#val) [![](w/page209a.gif)][P209] | [N3](#n3)<br> [![](w/page231a.gif)][P231] |
 |  12 | [diagr][A12] [![](w/page84a.gif)][P84] | [N4](#n4) [![](w/page105a.gif)][P105] | [diagr][C12] [![](w/page126a.gif)][P126] | [diagr][D12] [![](w/page145a.gif)][P145] | [diagr][E12] [![](w/page167a.gif)][P167] | [diagr][F12] [![](w/page190a.gif)][P190] | [diagr][G12] [![](w/page210a.gif)][P210] | [diagr][H12] [![](w/page232a.gif)][P232] |
 |     | **A** | **B** | **C** | **D** | **E** | **F** | **G** | **H** |
-|  13 | [N3](#n3) [![](w/page85a.gif)][P85] | [N3](#n3) [![](w/page106a.gif)][P106] | [N3](#n3) [![](w/page128a.gif)][P128] | [N3](#n3) [![](w/page147a.gif)][P147] | [N3](#n3) [![](w/page169a.gif)][P169] | [N3](#n3) [![](w/page192a.gif)][P192] | [N3](#n3) [![](w/page211a.gif)][P211] | [N3](#n3)<br> [![](w/page234a.gif)][P234] |
+|  13 | [N3](#n3), [3 pairs](#a13joins) [![](w/page85a.gif)][P85] | [N3](#n3) [![](w/page106a.gif)][P106] | [N3](#n3) [![](w/page128a.gif)][P128] | [N3](#n3) [![](w/page147a.gif)][P147] | [N3](#n3) [![](w/page169a.gif)][P169] | [N3](#n3) [![](w/page192a.gif)][P192] | [N3](#n3) [![](w/page211a.gif)][P211] | [N3](#n3)<br> [![](w/page234a.gif)][P234] |
```

Comment:
I don't think the link to N3 is needed any more.

```suggestion
|  13 | [3 pairs](#a13joins) [![](w/page85a.gif)][P85] | [N3](#n3) [![](w/page106a.gif)][P106] | [N3](#n3) [![](w/page128a.gif)][P128] | [N3](#n3) [![](w/page147a.gif)][P147] | [N3](#n3) [![](w/page169a.gif)][P169] | [N3](#n3) [![](w/page192a.gif)][P192] | [N3](#n3) [![](w/page211a.gif)][P211] | [N3](#n3)<br> [![](w/page234a.gif)][P234] |
```

A4 was mentioned in the user group in a pattern by Pierre Fouché. See also http://dentelles-de-javoison.over-blog.com/

Reply:
agreed